bin/oneoff/data_fix/fix_bad_school_info_in_users

```diff
@@ -64,8 +75,10 @@ CSV.open(csv_file, 'w') do |csv|
 
     users.each do |user|
       csv << [user.id.to_s, user.school_info_id.to_s]
-      if ACTUALLY_UPDATE
-        user.update!(school_info_id: nil)
+      if options[:actually_update]
+        user.last_seen_school_info_interstitial = DateTime.now + options[:interstitial_days]
```

Comment:
Why are we setting it to today (or the future)? Is this just to give them at least a week before [asking again](https://github.com/code-dot-org/code-dot-org/blob/8d961a3524e323e867e2d42e31a16ced3a8f4691/lib/cdo/school_info_interstitial_helper.rb#L16)?

Reply:
Marina asked that we delay the popup until after the school lookup is added. However, they need to fix the data now in order to run the other script that sets the school_info_id based on past PD enrollments.

Reply:
Gotcha. We will (obviously) need to make sure we add the school lookup before 7 days from whenever we set these to.

Reply:
We where going to future date it, 35 days from now on November 15th. With the future date the number of days is negative and the check will return false, which means don't show.